tiledb/ml/models/tensorflow_keras.py

```diff
@@ -322,81 +318,67 @@ def _serialize_optimizer_weights(self, include_optimizer: bool = True) -> bytes:
         else:
             return b""
 
-    def _load_custom_subclassed_model(self, model, model_array):
-
-        if "keras_version" in model_array.meta:
-            original_keras_version = model_array.meta["keras_version"]
-            if hasattr(original_keras_version, "decode"):
-                original_keras_version = original_keras_version.decode("utf8")
-        else:
-            original_keras_version = "1"
-        if "backend" in model_array.meta:
-            original_backend = model_array.meta["backend"]
-            if hasattr(original_backend, "decode"):
-                original_backend = original_backend.decode("utf8")
-        else:
-            original_backend = None
 
-        # Load weights for layers
-        self._load_weights_from_tiledb(
-            model_array[:], model.layers, original_keras_version, original_backend
+@staticmethod
+def _load_custom_subclassed_model(model, model_array):
```

Comment:
Either the indentation is wrong here or `@staticmethod` should go if this is intended to be a plain function (same for `_load_weights_from_tiledb`)

Reply:
Yeap I just did a check. Thnx